### PR TITLE
Add compatibility to symfony 5 and php 7.4

### PR DESCRIPTION
**Pull request by alexander-schranz**
related to #791, #792, #788

### CODE REVIEW

**Comment by sadikoff** [comment]
Any updates on it?

File: src/Selector/Xpath/Escaper.php [hunk 1]
```diff
@@ -47,6 +47,6 @@ public function escapeLiteral($s)
             }
         }
 
-        return sprintf('concat(%s)', implode($parts, ','));
+        return sprintf('concat(%s)', implode(',', $parts));
```
Comment (aik099) [inline src/Selector/Xpath/Escaper.php]: This is already done as part of #792.

**Comment by aik099** [comment]
@stof , changes made here is a variation of multi-PHPUnit version support, that was done prior to any decision reached in #787.

**Comment by stof** [comment]
This PR is not about supporting PHP 7.4 or Symfony 5 (which are both already supported without these changes), but about supporting PHPUnit 9 (due to `symfony/phpunit-bridge` selecting it by default).
I refactored our CI setup to have better control over the PHPUnit version (so that we decide when we update it, not Symfony deciding it for us). Using PHPUnit 9 will be done only after dropping support for older PHP versions (which is my next task on Mink now that releases are done), which will be much simpler than what is done here. So I'm closing this PR.

**Pull request closed without merging**